verticapy/performance/vertica/collection/profile_import.py

```diff
@@ -141,11 +210,92 @@ def _create_schema_if_not_exists(self) -> None:
 
     def _create_tables_if_not_exists(self) -> None:
         all_tables = getAllCollectionTables(
-            target_schema=self.target_schema, key=self.key
+            target_schema=self.target_schema, key=self.key, version=self.bundle_version
         )
         for ctable in all_tables.values():
             self.logger.info(f"Running create statements for {ctable.name}")
             table_sql = ctable.get_create_table_sql()
             proj_sql = ctable.get_create_projection_sql()
             _executeSQL(table_sql, method="fetchall")
             _executeSQL(proj_sql, method="fetchall")
+
+    def _unpack_bundle(self) -> Path:
+        """
+        Unpacks the bundle into a temp directory
+
+        Returns
+        --------
+        Returns a path
+        """
+
+        self.tarfile_obj = tarfile.open(self.filename, "r")
+        names = "\n".join(self.tarfile_obj.getnames())
+        print(f"Files in the archive: {names}")
+        # There are other ways to generate a
```

Comment:
Is the comment complete?

Reply:
oops, this is incomplete, thanks for catching that. I'll add the text that is missing

Reply:
Instead of fixing the comment, I did what the comment would have suggested, which is to make a directory unique by using a timestamp.